miner/builder_test.go

```diff
@@ -0,0 +1,97 @@
+package miner
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/consensus/clique"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuilder_AddTxn_Simple(t *testing.T) {
```

Comment:
Can we add `t.Parallel()` to the top of this test (idem for the others)?  Our test suite is quite lengthy as it is and I'm keen to cut down on its execution time where possible.